deobfuscator/src/main/java/net/runelite/deob/deobfuscators/UnusedParameters.java

```diff
@@ -298,7 +298,7 @@ public void run(ClassGroup group)
 			execution.run();
 
 			i = this.processUnused(execution, group);
-			System.out.println("PASS " + pnum++ + " " + i);
+			logger.info("PASS " + pnum++ + " " + i);
```

Comment:
I think this is just a debug statement I left in, I would remove it

Reply:
Addressed in 666063e